testing/test_doctest.py

```diff
@@ -129,6 +129,33 @@ def test_multiple_patterns(self, testdir):
             '*1 passed*',
         ])
 
+    @pytest.mark.parametrize(
+        '   test_string,    encoding',
+        [
+            (u'foo',         'ascii'),
+            (u'öäü',         'latin1'),
+            (u'öäü',         'utf-8')
+        ]
+    )
+    def test_encoding(self, testdir, test_string, encoding):
+        """Test support for doctest_encoding ini option.
+        """
+        testdir.makeini("""
+            [pytest]
+            doctest_encoding={0}
+        """.format(encoding))
+        doctest = u"""
+            >>> u"{}"
```

Comment:
The use of `{}` is not supported by `py26` and is breaking the CI, please change that to `{0}`, `{1}`, etc.